src/xclim/indices/_conversion.py

```diff
@@ -1319,6 +1319,58 @@ def _get_D_from_M(time):  # noqa: N802
     )
 
 
+# TODO: How can I check exact units for inputs
+# @declare_units(
+#     net_radiation="[MJ m-2 day-1]",
+#     tas="[degC]",
+#     wind="[m s-1]",
+#     es="[kPa]",
+#     ea="[kPa]",
+#     delta_svp="[kPa degC-1]",
+#     gamma="[kPa degC]",
+#     G="[MJ m-2 day-1]",
+# )
```

Comment:
```suggestion

@declare_units(
    net_radiation="[radiation]",
    tas="[temperature]",
    wind="[speed]",
    es="[pressure]",
    ea="[pressure]",
    delta_svp="[kPa degC-1]",
    gamma="[kPa degC]",
    G="[radiation]",
)
```

Reply:
nice I forgot that trick

Reply:
The only thing I dont get is how to provide scalars e.g. if I do `G="0 MJ m-2 day-1"` I get an obvious cant subtract string from DataArray error.?

Reply:
That's where `convert_units_to` comes in!

`convert_unitS_to("0 MJ m-2 day-1", "MJ m-2 day-1")` returns 0, as a float, which can be used in further arithmetics.

Reply:
Gotcha done!